src/utils/context.js

```diff
@@ -0,0 +1,19 @@
+import { createContext, useState } from "react"
```

Comment:
Nice that you used Context API. I have never used it. Would love to pair to learn more about how to implement it.